### ISSUE TITLE
Trailing spaces in Mundo window

### ISSUE THREAD

**Issue by Xblow** [simnalamburt/vim-mundo#101]
I was wondering if it's possible to **remove trailing spaces** in the Mundo preview window. 
I usually mark extra whitespace and I noticed there is unnecessary whitespace in Mundo tree viewer.
You can reproduce it with
```
highlight ExtraWhitespace ctermbg=brown guibg=brown
match ExtraWhitespace /\s\+$/
```
with mundo window open.
Over, it's an amazing plugin, thank you! :+1:

**Comment by simnalamburt** [comment]
I agree that this is an issue and I think it's possible to fix. I'll try to strip the whitespaces or don't append the whitespaces in the first place later.
